source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,274 @@
+.. _cpp-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, text search, efficiency
+
+Overview
+--------
+
+The MongoDB Atlas Search feature enables you to perform full-text
+searches on collections hosted on Atlas. Before you can perform Atlas
+Search queries, you must create indexes that specify which
+fields to index and how they are indexed.
+
+To learn more about Atlas Search, see the :atlas:`Atlas Search Overview
+</atlas-search/atlas-search-overview/>`.
+
+You can use the following methods on a ``mongocxx::search_index_view`` instance
+to manage your Atlas Search indexes:
+
+- ``create_one()``
+- ``create_many()``
+- ``list()``
+- ``update_one()``
+- ``drop_one()``
+
+To get a ``search_index_view`` to your collection, 
+call the ``search_indexes()`` method on your ``mongocxx::collection`` instance.
+
+.. note:: Atlas Search Index Management is Asynchronous
+
+   The {+driver-long+} manages Atlas Search indexes asynchronously. The
+   library methods described in the following sections return the server
+   response immediately, but the changes to your Search indexes take
+   place in the background and might not complete until some time later.
+
+The following sections provide code examples that demonstrate how to use
+each Atlas Search index management method.
+
+.. _cpp-atlas-search-index-create:
+
+Create a Search Index
+---------------------
+
+To create a single Atlas Search Index on a collection, call the ``create_one()`` method on a ``mongocxx::search_index_view`` instance and pass in a ``mongoxcc::search_index_model`` 
```

Comment:
```suggestion
To create a single Atlas Search index on a collection, call the ``create_one()`` method on a ``mongocxx::search_index_view`` instance and pass in a ``mongoxcc::search_index_model`` 
```

Reply:
Is there a reason why this instance of "index" should be capitalized? I've found that the Atlas documentation always uses a lowercase "index." (e.g. [Create an Atlas Search Index](https://www.mongodb.com/docs/atlas/atlas-search/create-index/))